prototype_source/inductor_with_x86_cpu_as_backend_of_pt2e.rst

```diff
@@ -0,0 +1,186 @@
+TorchInductor with X86 CPU as backend of Quantization in PyTorch 2.0 Export
```

Comment:
please update the name to align better with existing names, see https://github.com/pytorch/tutorials/pull/2551/files#r1322237340

Reply:
Thanks and changed.